mergify_engine/context.py

```diff
@@ -240,11 +240,11 @@ def _get_consolidated_data(self, name):
         # tracked, and on big projects it can reach the rate limit very
         # quickly.
         # NOTE(sileht): Not handled for now: cancelled, timed_out, or action_required
-        elif name == "status-success":
+        elif name in ("status-success", "check-success"):
```

Comment:
if we replace it at parsing, is there's a chance we still encounter it here?

Reply:
For Jinja templating, I will try to find a way to do the same I have done with pyparing, with Jinja. It would be cleaner.